lib/scrub/autoscrub.rb

```diff
@@ -84,13 +92,18 @@ def run
       Services.scrub_logger.info(
         "Output file moved to #{@output_struct.member_ready_to_load.to_path}"
       )
+      # Move file and store new location in array.
+      @out_files << File.join(
+        @output_struct.member_ready_to_load,
+        File.split(out_file_path).last
+      )
     end
 
     private
 
     def count_file_lines
       # zcat -f works as plain cat if @path is not in gzip format.
-      `zcat -f #{@path} | wc -l`.match(/^(\d+)/)[0].to_i
+      `zcat -f "#{@path}" | wc -l`.match(/^(\d+)/)[0].to_i
```

Comment:
Consider doing this in Ruby -- not as elegant for opening a maybe-gzipped file, although we could extract that out as a helper somewhere (we're doing it in a couple other places in the code base already e.g. in lib/concordance_validation). I think this should do it:

```ruby
require 'zlib'

if @path.end_with?(".gz") then
  Zlib::GzipReader.open(@path)
else
  File.open(@path)
end.count
```

Reply:
Added and plugged in `Utils::LineCounter`.